Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -175,10 +188,12 @@ struct SegmentedSwitchPreviews: PreviewProvider {
             VStack(spacing: .large) {
                 SegmentedSwitch(
                     "Gender\nmultiline",
-                    firstOption: "Male with long name",
-                    secondOption: "Female with much longer name",
-                    selectedIndex: value
-                )
+                    firstOption: 1,
+                    secondOption: 2,
+                    selection: value
+                ) {
+                    $0 == 1 ? "Male with long name" : "Female with much longer name"
```

Comment:
This way of providing content seems weird. Do we use it somewhere else? 

Maybe we could introduce custom builders - that variant would look like this:

```
SegmentedSwitch("Gender", selectedValue: 2) {
   Segment(1, "Male")
   Segment(2, "Female")
}
```

Reply:
The way `Picker` does it is by assigning `.tag`s to the individual segments. Since the primary way we want to use this is not to specify custom views, but only strings (while retaining a way to set custom views), I figured this made more sense. Usually you would use an enum, not `Int`, so you would have a `switch` here.

I actually tried what you propose when I attempted to update `Tabs`, but the types aren't inferred in the builder for some reason, so you can't just use the `.caseName` notation.

Since this is limited to two options, another way to do it would be to pass the strings directly as parameters, eliminating this closure.